src/client/connection.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @hidden
```

Comment:
This is entirely my ignorance, but what does this do?

Reply:
Hides the file from `typedoc`.  I'm guessing it was left in from another project, but, should remain as it will be needed eventually when docs come.

Reply:
It was copied from the layouts project, but I left it there intentionally since that file is all the client stuff we don't want in the docs.